lib/terraform.nix

```diff
@@ -21,6 +22,8 @@ rec {
     type = "app";
     program = toString (pkgs.writers.writeBash command ''
       pushd ${toString subdir} >/dev/null
+      export VAULT_ADDR='${vault-addr}'
```

Comment:
Also, if we can dynamically check if Vault plugin is used, Vault login can be made optional